lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require 'menu'
+
+class Order
+  def initialize(*choice, quantities, user_total)
+    @choice = choice
+    @quantities = quantities
+    @user_total = user_total
+    @menu = Menu.new
+  end
+
+  def calculate_price
```

Comment:
I'm not sure about your approach to entering dishes and quantities, but this is a nice way to handle them in the method